### DISCUSSION TITLE
Lightspeed Point blank

### DISCUSSION THREAD

**Discussion by CrazyDibber** [3kh0/ext-remover#456] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
I realized that by clicking the licenses link and opening it in a new window, you could spam esc to get a lightspeed javascript-enabled page. This could be implemented into point-blank (new)

**Comment by Aka-but-nice** [comment]
Yea this is possible, and it may load into a blank page, but one of the problems is, it may say when you use javascript:alert('Test') it might say "This page says" to test on the blank page thing, make a bookmarklet wih javascript:alert('Test') and use it on the blank page.

**Reply by CrazyDibber** [reply]
i got:

**Lightspeed Filter Agent**

**Test**

**Reply by Aka-but-nice** [reply]
OH MY GOSH

**Reply by Aka-but-nice** [reply]
Give me a sec

**Reply by ghost** [reply]
> I realized that by clicking the licenses link and opening it in a new window, you could spam esc to get a lightspeed javascript-enabled page. This could be implemented into point-blank (new)

it don't work

**Reply by Aka-but-nice** [reply]
> i got:
> 
> **Lightspeed Filter Agent**
> 
> **Test**

@CrazyDibber BRo send me a pic

**Reply by Aka-but-nice** [reply]
> i got:
> 
> **Lightspeed Filter Agent**
> 
> **Test**

Would you send me a pic

**Reply by funnypenguin100** [reply]
Hmm... interesting concept, however, on a school computer with all javascript disabled this does not appear to work for me. The escape indeed loads into a blank page, however; typing in javascript:alert('Test') into the search bar doesn't seem to work... I'll continue testing

**Reply by Aka-but-nice** [reply]
No, you don't have too, this is confirmed not to work.

**Reply by phillipbruh** [reply]
no it does work, i can run alert
![Screenshot 2023-03-09 11 15 21 AM](https://user-images.githubusercontent.com/95700058/224085285-3f0cd96a-7d1a-42b6-8a40-953d68560ad3.png)

**Reply by Aka-but-nice** [reply]
I know it works, but you can't alert localStorage try using ```
javascript:alert(localStorage);```
And get back to me

**Comment by Crazygird** [comment]
@Aka-unblock

**Reply by Aka-but-nice** [reply]
?

**Comment by Crazygird** [comment]
picture:
<img width="1665" alt="Screenshot 2023-01-30 at 4 34 07 PM" src="https://user-images.githubusercontent.com/95974074/215628395-06cbec96-71d7-4574-9f43-ec0e3e01845f.png">

**Comment by ghost** [comment]
[Screen recording 2023-01-30 6.38.59 PM.webm](https://user-images.githubusercontent.com/115483649/215628961-b94034b7-dbb8-4f62-8367-b44b016aae6b.webm)


it doesnt work

**Reply by ghost** [reply]
idk why you can't see me click on the extensions

**Reply by ghost** [reply]
and im pressing escape

**Comment by ghost** [comment]
does this work on latest version

**Reply by Crazygird** [reply]
I'm on v109

**Reply by CrazyDibber** [reply]
open in new **window** btw

**Comment by Crazygird** [comment]
heres a tutorial. 


https://user-images.githubusercontent.com/95974074/215630217-52b9731b-588c-47a4-b62a-f6d21bd9fa7b.mov


heres y

**Reply by Crazygird** [reply]
u have to spam esc when u open

**Reply by Aka-but-nice** [reply]
@CrazyDibber

**Comment by ghost** [comment]
IT WORKED!

**Comment by ghost** [comment]
![image](https://user-images.githubusercontent.com/115483649/215631362-4764a502-cd6a-43e0-99ed-4daf94ea896c.png)

**Comment by Crazygird** [comment]
cool! I'm excited to see how far this will go if it's able to be implemented with point blank!

**Reply by Aka-but-nice** [reply]
Alr, bet, Imma have Bypassi and me work on this.

**Comment by Crazygird** [comment]
@3kh0

**Reply by Aka-but-nice** [reply]
tf, why yall pinging him, he has never said anything, he doesn't know how to make a Point blank launcher

**Comment by Crazygird** [comment]
bruh, who does then?

**Comment by Crazygird** [comment]
oh, just read past comment

**Reply by Crazygird** [reply]
> Alr, bet, Imma have Bypassi and me work on this.

**Comment by Crazygird** [comment]
@Aka-unblock anything new?

**Reply by Aka-but-nice** [reply]
Well, no, Bypassi said it wouldn't work, to test, use either javascript:opener.chrome.extension.getBackgroundPage().close()
or javascript:chrome.extension.getBackgroundPage().close()

**Comment by Crazygird** [comment]
where and how do I run this?

**Reply by Aka-but-nice** [reply]
On the supposedly light-speed blank page, or the page you showed us earlier.

**Comment by Crazygird** [comment]
what results should I be getting

**Reply by viperbot2000** [reply]
did it work?

**Reply by Crazygird** [reply]
im not sure what should be happening

**Comment by Crazygird** [comment]
@Aka-unblock  I JUST FOUND SOMETHING INSANE! Basically, the html of the "licence" link targets parent. If you hold ctrl+shift and click the link it opens it in the same window! This will make it possible to spam esc to get a javascript in-window page!!!!!!

**Reply by Crazygird** [reply]
hold up, opener doesn't work.

**Reply by Aka-but-nice** [reply]
> hold up, opener doesn't work.

As expected, it won't work, it has to exactly specify whether if it's going to use rel="opener", or target="_blank"

**Reply by Aka-but-nice** [reply]
Try functions without opener

**Reply by Aka-but-nice** [reply]
> hold up, opener doesn't work.

Use javascript:alert(localStorage)

**Reply by Crazygird** [reply]
nothing happens when I run  ```javascript:alert(localStorage)```

**Reply by Crazygird** [reply]
but on other sites it says ```[object Storage]```

**Reply by Aka-but-nice** [reply]
> nothing happens when I run `javascript:alert(localStorage)`

Yep, as expected, this shall not be possible.

**Reply by Aka-but-nice** [reply]
> nothing happens when I run `javascript:alert(localStorage)`

That was the number 1 test, if that didn't pass, it is not possible.
